codec-http/src/main/java/io/netty/handler/codec/http/multipart/DiskFileUpload.java

```diff
@@ -147,8 +147,7 @@ protected String getBaseDirectory() {
 
     @Override
     protected String getDiskFilename() {
-        File file = new File(filename);
-        return file.getName();
+        return "upload";
```

Comment:
if this is a fixed value doesn't that mean that existing users may observe a change in behavior and collision in file names if they are not using `getBaseDirectory()`, or `getBaseDirectory()` collides?